isort/info.yaml

```diff
@@ -0,0 +1,47 @@
+---
+enabled: true
+name: isort
+version: 4.3.21
```

Comment:
Can you add a `v` in front of this? It wasn't super intentional, but I think all other Restylers are doing that now, so I'd like to keep doing it until I can give it more thought and decide if I actually care or not :smile:

Reply:
I agree with that @pbrisbin! Thanks! 😉